fleet/api/responses.go

```diff
@@ -166,3 +171,20 @@ func (s validateAgentGroupRes) Headers() map[string]string {
 func (s validateAgentGroupRes) Empty() bool {
 	return false
 }
+
+type validateAgentRes struct {
+	Name      string 		 `json:"name"`
```

Comment:
Use the reformat code, you can configure it on intellij before the commits (Search for reformat code on slack)